### DISCUSSION TITLE
Avoid increasing the package file version (package.json or package-lock.json) during development phase

### DISCUSSION THREAD

**Discussion by AntoniaGross** [renovatebot/renovate#29920] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

None

### Please tell us more about your question or problem

Hello,
during the development phase the renovate bot is running regularly. A side effect is that the package version or project version in the package files is always increased as soon as dependency version updates have been detected. 
Example package.json:

![PackageVersion](https://github.com/renovatebot/renovate/assets/160839170/d63c8f9f-c1a1-433b-916c-23aa991cee7d)

The configuration file of renovate:
![renovateConfig](https://github.com/renovatebot/renovate/assets/160839170/c439bda3-bb23-45b3-a84b-acef3373860e)

Is it possible to configure it, so that the package.json version is not updated automatically?

Thank you for your feedback and support in advance.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Remove your bumpVersion=patch config

**Reply by AntoniaGross** [reply]
Thank you for your quick support. I would like to ask one more question: If I remove bump version, the application version is no longer incremented like you suggested. However, only the package-lock.json is taken into account and the synchronization with package.json is lost. So I can't have both?

**Comment by AntoniaGross** [comment]
Thank you for your quick support. I would like to ask one more question: If I remove bump version, the application version is no longer incremented like you suggested. However, only the package-lock.json is taken into account and the synchronization with package.json is lost. So I can't have both?
